executor/README.md

```diff
@@ -166,45 +194,57 @@ Then, to install the executor executable (`cook-executor`) and all dependencies,
 ./setup.py install
 ```
 
+Alternatively, `pyinstaller` can be used to build a single executable that is distributed via a URI. For example, to build the executable, run the following command from the `executor` folder:
```

Comment:
I haven't heard of pyinstaller. Is it a well-known tool? 

Can you add a link to it in the doc?